addons/stock/models/res_config_settings.py

```diff
@@ -59,6 +59,8 @@ def _onchange_group_stock_multi_locations(self):
     def _onchange_group_stock_production_lot(self):
         if not self.group_stock_production_lot:
             self.group_lot_on_delivery_slip = False
+            if self.env['product.product'].search_count([('tracking', '!=', 'none')], limit=1):
```

Comment:
limit=1 + search_count() = Traceback
Please change this error, use search and limit

Reply:
"limit" was added in 16.0 and it works fine for me, can you say something more?
I removed "limit" from 15.0 & 15.2 with another PR https://github.com/odoo/odoo/pull/140420

Reply:
Thanks 😄